wordpress-mobile/WordPress-Android
# Accessibility : Posting Stats Improvement v2

Fixes #10975 
## Findings
The Posting Stats was inaccessible when using TalkBack. This improvement makes it more accessible for visually impaired users who are navigating the stats functionality. 

## Solution
The solution for this is to highlight the views by grouping them by the days that share the same ranking([This solution was discussed in detail on this PR](https://github.com/wordpress-mobile/WordPress-Android/pull/10980#pullrequestreview-334602495)). Once this is done, the user is able to hear the days with the highest views as soon as they interact with the component and then can continue to cycle through the rest of activity with subsequent clicks. 

Before | After 
--------|-------
[Before Video ](https://drive.google.com/open?id=18ErO8-vpZxLuxucriVecOq218nEcmaSK) |   [After Video    ](https://drive.google.com/file/d/19AwFLbvKmtyl16pL9nG3BLGEDjRnKA_h/view?usp=sharing)

## Testing

1. Enable TalkBack. 
2. Navigate to the Stats page.
3. Ensure you are on the Insights Tab. 
4. Select each Month under "Posting Activity" and follow the instructions to hear the announcement.

Tap To Hear Day Stats | Stat Announced 
--------|-------
<img src="https://user-images.githubusercontent.com/1509205/71115552-2fe14d80-21a0-11ea-8fd9-427a58b41d14.png" width="320">        |       <img src="https://user-images.githubusercontent.com/1509205/71901094-2c047380-312d-11ea-8d24-af67d40ef3e3.png" width="320">

All Stats Announced | No Stats Available 
--------|-------
<img src="https://user-images.githubusercontent.com/1509205/71115354-bea19a80-219f-11ea-9193-8bcec2daeb6c.png" width="320">   |    <img src="https://user-images.githubusercontent.com/1509205/71115439-e2fd7700-219f-11ea-8c70-34569e4c7c2e.png" width="320">

## Reviewing 
Only 1 reviewer is needed but anyone can review. 

## Submitter Checklist
